docker/Dockerfile.osmo-smsc5

```diff
@@ -0,0 +1,35 @@
+FROM debian
```

Comment:
osmo-smsc4 and osmo-smsc5 sound like different SMSC versions. If you copy the VM in both cases then there should be no difference in this file and the only difference will be in docker/docker-image.sh?